README.md

```diff
@@ -126,6 +126,19 @@ You will now also need to update your `.env` file to make sure you're using this
 
 If you need to reset this database, rerun step 2 (with `dropdb foundation` as first command) through 5 to get back to a clean copy of the production database.
 
+
+### Resolving conflicting Django migrations
+
+**AKA: What to do when a migration lands before yours**
+
+- Create a new, separate local instance of `foundation` per "Setup steps" above.
```

Comment:
Hmm, this doesn't quite feel required. You should be to do this in your branch without setting up a new copy.

Reply:
meh, let's just merge anyway =D